spk/java-11-openjdk/Makefile

```diff
@@ -0,0 +1,60 @@
+SPK_NAME = java-11-openjdk
+SPK_VERS = 11.0
+SPK_REV = 1
+SPK_ICON = src/$(SPK_NAME).png
+
+DEPENDS = cross/$(SPK_NAME) cross/cups
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = The OpenJDK 11 runtime environment. Version 11.0.12+7
+RELOAD_UI = no
```

Comment:
RELOAD_UI is deprecated an can be removed.

Reply:
Removed